llvm/test/Verifier/AMDGPU/kernel-recursivecall.ll

```diff
@@ -0,0 +1,9 @@
+; RUN: not llvm-as %s -o /dev/null 2>&1 | FileCheck %s
```

Comment:
-disable-output is canonical way to disable output, also this does not require AMDGPU